src/modules/map/components/OlMap.js

```diff
@@ -21,6 +21,8 @@ export class OlMap extends BaElement {
 
 	constructor() {
 		super();
+		const { ShareService } = $injector.inject('ShareService');
```

Comment:
Our Variable names should start with lower case. We can rename within the destructuring expression with
`const { ShareService : shareService } =  `